src/app/core/feedback/feedback.routes.js

```diff
@@ -68,13 +66,22 @@ const router = express.Router();
  *                 type: 'no-login'
  *                 message: 'User is not logged in'
  */
-router.route('/feedback')
+router
+	.route('/feedback')
 	.post(user.has(user.requiresLogin), feedback.submitFeedback);
 
-router.route('/admin/feedback')
-	.post(user.hasAdminAccess, feedback.search);
+router.route('/admin/feedback').post(user.hasAdminAccess, feedback.search);
+
+router
```

Comment:
Could we add swagger docs for the added methods?

Reply:
@jrassa have you found a good way to separate the admin-facing Swagger docs from the user-facing docs? It's not a security risk for exposing those admin route docs to folks, but seems like it would clutter things up for non-admin users (as @tzfx suggested a while ago).

Reply:
Not really.  I have been adding an 'Admin' tag to some operations, but that just results in them being added twice.  Once under each tag.  Reading now, it looks like we can enable filtering by tags which would help some.

There are also ways to extend/customize the swagger ui.  That could be an option to explore as well.

Reply:
- [ ] Add Swagger Docs for new endpoints

Reply:
I've added swagger docs for the two new endpoints. I figure we can sort out how to tag/show them properly once we figure out how we want to handle that across the board.

Reply:
Swagger docs look good.  I pushed a small update to allow setting the feedbackId path parameter.